src/importlinter/domain/helpers.py

```diff
@@ -31,14 +32,23 @@ def pop_imports(
             importer=import_to_remove.importer.name, imported=import_to_remove.imported.name
         )
         if not import_details:
-            raise MissingImport(f"Ignored import {import_to_remove} not present in the graph.")
+            missing_imports.append(import_to_remove)
+            continue
 
         graph.remove_import(
             importer=import_to_remove.importer.name, imported=import_to_remove.imported.name
         )
 
         removed_imports.extend(import_details)
 
+    if missing_imports:
+        # Sort the output to make it more helpful
+        missing_import_strings = sorted([str(i) for i in missing_imports])
```

Comment:
Not a blocker but we could make this just use a generator, i.e. `sorted(str(i) for i in missing_imports)`